### DISCUSSION TITLE
The game doesn't start

### DISCUSSION THREAD

**Discussion by Bleydeolbras** [tsunamods-codes/7th-Heaven#184] [Q&A] [answered]
![test](https://github.com/tsunamods-codes/7th-Heaven/assets/161249506/7939c507-537b-4636-bd01-3fbfc0e9e468)
Hello, sorry I'm French and I translate with Google to save time. After searching your forums and Discord I can't find my answer. I just installed 7th Heaven and FFNX for Steam but when I launch the game I get this message. I haven't installed any mods yet and am running the game without mods. Can you help me?

**Comment by julianxhokaxhiu** [comment, marked as answer]
Please consider using Discord support channels to troubleshoot these issues. Cheers!
